scripts/Tools/baseline_gen_comp

```diff
@@ -1,265 +0,0 @@
-#!/bin/bash
```

Comment:
The baseline_gen_comp script provided critical functionality. I don't see where this is provided now. In particular, this allowed you to run all baseline comparisons or generations on a given test suite after the fact. Some typical workflows were:

1. Running a test suite without the -generate option, then later (after you determine things look good), run baseline_gen_comp to generate baselines in the appropriately-named baseline directory

2. Running a test suite with comparison against some baseline tag 'foo', but then later also comparing that test suite with some different baseline tag 'bar'.

3. Running a test suite without any baseline comparisons, and doing the comparisons in a separate step later. This is particularly useful when running two test suites at once from different code bases, then comparing them after they both finish.

Is it possible to achieve this functionality with the new scripts? If not, I would ask that this functionality be restored before this PR comes to master.